modules/bat_event/bat_event.admin.inc

```diff
@@ -449,11 +453,6 @@ function bat_event_edit_form_submit(&$form, &$form_state) {
   // Add the event to $form_state to be altered by other submit handlers.
   $form_state['event'] = $event;
 
```

Comment:
@istos looks like this code was just kind of... there. so I nuked it. But if I'm missing some function it was performing lmk ;)